Makefile

```diff
@@ -198,6 +198,30 @@ else
 	IGNORE:=$(shell rm -f imports/boards_imports.go)
 endif
 
+# Playbooks
+BUILD_PLAYBOOKS_DIR ?= ../mattermost-plugin-playbooks
```

Comment:
Nit: Maybe move this section after L94 where the rest of the declarations for webapp, focalboard are? And also move the `LDFLAGS +=` line with others. 0/5